packages/wrangler/package.json

```diff
@@ -37,6 +37,7 @@
   ],
   "dependencies": {
     "esbuild": "0.14.14",
+    "@cloudflare/workers-wasi": "0.0.4",
```

Comment:
Do we need to import this as a dependency? This is only used in the glue JS code that is generated, and it would be nicer if the developer could control the version of this dependency without wrangler pinning it.

Might be worth using the package manger that @petebacondarwin is introducing in #373?

Not a dependency expert, so also cc'ing @threepointone

Reply:
I'm not so sure we want to change this because for wasm there is no package.json, no wrangler.toml etc. that the consumer provides, where would they define which version they want to pin?

I'm open to exposing something like that but I think we should start with keeping the version controlled by wrangler so we can test end to end (with wrangler dev/publish etc.) when we bump the wrangler pinned version

Distributing it this way means theres no special thing that we have to worry about for wasi specifically, any case where wrangler is installed the module will be correctly installed and we can guarantee that the on-the-fly bundling will work.  Since the module is ~75k and likely to grow smaller over time (and has no prod deps) I don't think there is much downside

Reply:
Good points.. been thinking so long in a JS world, glossed over not having the package.json et al.